File: src/MyBikingApp/app/src/main/java/com/example/gpstrackingdemo/Point.java
package com.example.gpstrackingdemo;

import android.location.Geocoder;

public class Point {
    private double latitude;
    private double longitude;
    private String address;

    public Point(double latitude, double longitude, String address){
        this.latitude = latitude;
        this.longitude = longitude;
        this.address = address;
    }

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }
}

File: src/MyBiking.Entity/Models/Point.cs
namespace MyBiking.Entity.Models
{
    public class Point
    {
        public int Id { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Address { get; set; }
        public Ride Ride { get; set; }
        public int RideId { get; set; }
    }
}
